agroal-test/src/test/java/io/agroal/test/basic/BasicConcurrencyTests.java

```diff
@@ -53,6 +53,7 @@ public class BasicConcurrencyTests {
     @BeforeAll
     static void setup() {
         registerMockDriver();
+        Utils.windowsTimerHack();
```

Comment:
Maybe only execute on windows? (`System.getProperty("os.name").startsWith("Windows")`)

Reply:
I added a Utils.isWindowsOS() method to check if the OS is Windows. Perhaps later this should be done a bit nicer if and when the necessity arises.